internal/kms/repository_database_key.go

```diff
@@ -44,7 +44,10 @@ func createDatabaseKeyTx(ctx context.Context, r db.Reader, w db.Writer, rkvWrapp
 	if len(key) == 0 {
 		return nil, nil, errors.New(ctx, errors.InvalidParameter, op, "missing key")
 	}
-	rootKeyVersionId := rkvWrapper.KeyID()
+	rootKeyVersionId, err := rkvWrapper.KeyId(ctx)
+	if err != nil {
+		return nil, nil, errors.Wrap(ctx, err, op, errors.WithMsg("unable to get key id"))
```

Comment:
a bit inconsistent with other error msgs for this.. I think, you were using something like "unable to lookup root key"